blockchain/vm/jump_table.go

```diff
@@ -106,7 +107,7 @@ func newConstantinopleInstructionSet() JumpTable {
 		constantGas:     params.ExtcodeHashGasConstantinople,
 		minStack:        minStack(1, 1),
 		maxStack:        maxStack(1, 1),
-		computationCost: params.ExtCodeHashComputationCostConstantinople,
+		computationCost: params.ExtCodeHashComputationCost,
```

Comment:
Doesn't have any problem before `IstanbulCompatibleBlock`?

Reply:
It is the revert of the computation cost modification which is applied at https://github.com/klaytn/klaytn/pull/958 :)

Reply:
I think yumiel set both ExtCodeHashComputationCostConstantinople and ExtCodeHashComputationCostEIP1884 to 1000 for the simplexity